### ISSUE TITLE
NTR: 2,8-dihydroxyadenine high in urine on HPLC (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8857]
New term request 2,8-dihydroxyadenine high in urine on HPLC (segal_120706192157): DHA

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000624
- label: Elevated urinary 2,8-dihydroxyadenine level
- parents: [Abnormal urinary purine level (HP:0033161)](https://hpo.jax.org/app/browse/term/HP:0033161)
- pmid: PMID:16613999
- def: The amount of 2,8-dihydroxyadenine in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 2,8-Dihydroxyadenine is a Purine metabolite usually not detectable in biofluids of normal individuals but is identified in Adenine phosphoribosyltransferase deficiency.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8857
- synonyms:

**Comment by MickeySegal** [comment]
There is some solubility, and one can measure soluble DHA or observe crystals.  
https://www.ncbi.nlm.nih.gov/books/NBK100238/
"_Surveillance_: Measurement of eGFR and urinary DHA excretion (or urine microscopy for assessment of DHA crystalluria) every 6-12 months; routine follow up to facilitate adherence to pharmacologic treatment at least annually... Urine microscopy for assessment of DHA crystalluria if direct DHA measurements not available... Urinary DHA measurements. The authors' group has recently developed a urinary DHA assay using ultra-high-performance liquid chromatography – tandem mass spectrometry (UPLC-MS/MS) [Thorsteinsdottir et al 2016], which is expected to facilitate both the clinical diagnosis and monitoring of pharmacotherapy of individuals with APRT deficiency [Edvardsson et al 2018]. Further clinical studies to examine the sensitivity and specificity of the assay are currently under way."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000803
- label: Elevated urinary 2,8-dihydroxyadenine level
- parents: [Abnormal urinary purine level (HP:0033161)](https://hpo.jax.org/app/browse/term/HP:0033161)
- pmid: PMID:22934314
- def: The amount of 2,8-dihydroxyadenine in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8857
- synonyms:
